File: server/plugins/MineParkCore/src/minepark/models/dtos/UserDto.php
<?php
namespace minepark\models\dtos;

class UserDto extends BaseDto
{
    public int $id;

    public string $name;

    public string $fullName;

    public ?string $email;

    public ?string $group;

    public ?string $people;

    public ?string $tag;

    public int $bonus;

    public int $minutesPlayed;

    public int $phoneNumber;

    public bool $vip;

    public bool $administrator;

    public bool $builder;

    public bool $realtor;

    public ?PlayerBanDto $ban;

    public string $joinedDate;

    public string $leftDate;

    public string $createdDate;

    public string $updatedDate;
}

File: mdc/DataCenter/MDC/Data/Dtos/UserDto.cs
using System;

namespace MDC.Data.Dtos
{
    public class UserDto : IdentifiedDto
    {
        public string Name { get; set; }

        public string FullName { get; set; }

        public string Group { get; set; }

        public string People { get; set; }

        public string Tag { get; set; }

        public int Bonus { get; set; }

        public int MinutesPlayed { get; set; }

        public long PhoneNumber { get; set; }

        public bool Vip { get; set; }

        public bool Administrator { get; set; }

        public bool Builder { get; set; }

        public bool Realtor { get; set; }

        public PlayerBanDto Ban { get; set; }

        public DateTime JoinedDate { get; set; }

        public DateTime LeftDate { get; set; }

        public DateTime CreatedDate { get; set; }

        public DateTime UpdatedDate { get; set; }
    }
}
